### PR TITLE
Small updates, improvements, and housekeeping to docs

### PR DESCRIPTION
**Pull request by joelhans**
##### Summary

A handful of docs improvements that have been sitting in my backlog.

- Add new collectors to supported collectors list.
- Fix Berkeley typos.
- Remove reference to OpenMetrics.
- Clarify that exporting sends metrics only from the present, not the entire internal database.
- Clarify how to use `touch` to create a new health entity file, otherwise `edit-config` throws an error.
- Update images with newest Netdata Cloud UI/nav.

##### Component Name

area/docs

### CODE REVIEW

**Review by aabatangle — APPROVED** [review]

File: docs/export/enable-connector.md [hunk 1]
```diff
@@ -44,9 +47,9 @@ Use the following configuration as a starting point. Copy and paste it into `exp
 Replace `my_opentsdb_http_instance` with an instance name of your choice, and change the `destination` setting to the IP
 address or hostname of your OpenTSDB database.
 
-Restart your Agent with `service netdata restart` to begin exporting to your OpenTSDB database. Because the
-Agent exports metrics as they're collected, you should start seeing data in your external database after only a few
-seconds.
+Restart your Agent with `service netdata restart` to begin exporting to your OpenTSDB database. The Netdata Agent
```
Comment (thiagoftsm) [inline docs/export/enable-connector.md:50]: Joel I remember I read some place that nowadays is better to use `systemctl` instead `service` to restart a service on Linuxes (@Ferroin , please, can you confirm this?). If this is truth we should suggest to use `systemctl` or to teach both.

Comment (joelhans) [reply docs/export/enable-connector.md:50]: Interesting! Yes, let's get confirmation from @Ferroin, but I was under the assumption that `service` is a wrapper that works on systems with both systemd and others, like `init.d`. So, potentially, `service` works on more systems than `systemctl`.

Comment (vlvkobal) [reply docs/export/enable-connector.md:50]: You are right. I thought that `service` is deprecated, but
> If there is no suitable legacy action found and COMMAND is one of actions specified in LSB Core Specification, input is redirected to the systemctl.

File: docs/monitor/configure-alarms.md [hunk 2]
```diff
@@ -75,13 +76,18 @@ your systems, containers, and applications work.
 Read Netdata's [health reference](/health/REFERENCE.md#health-entity-reference) for a full listing of the format,
 syntax, and functionality of health entities.
 
-To write a new health entity, use `edit-config` to create a new file inside of the `health.d/` directory.
+To write a new health entity into a new file, navigate to your [Netdata config directory](/docs/configure/nodes.md),
+then use `touch` to create a new file in the `health.d/` directory. Use `edit-config` to start editing the file.
```
Comment (thiagoftsm) [inline docs/monitor/configure-alarms.md:80]: @joelhans , please, add a short text to explain for the users that files inside `health.d`must have an extension `.conf`. Thank you!

**Review by aabatangle — DISMISSED** [review]

**Review by OdysLam — APPROVED** [review]

**Review by aabatangle — APPROVED** [review]

**Review by OdysLam — APPROVED** [review]

**Pull request merged**